### PR TITLE
test(no-tablets jobs): add no-tablets jobs weekly trigger

### PR DESCRIPTION
**Pull request by yarongilor**
Weekly trigger some jobs of 'no_tablets' folder.
	refs: #7655
refs: #7655

### Testing

- [ ]

### PR pre-checks (self review)

- [ ] I added the relevant `backport` labels
- [ ] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: jenkins-pipelines/master-triggers/sct_triggers/no_tablets_weekly_trigger.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?><project>
+  <actions/>
+  <description>Triggers Tablets Tests on weekly basis, every Saturday at 10:00
```
Comment (xtrey) [inline jenkins-pipelines/master-triggers/sct_triggers/no_tablets_weekly_trigger.xml:3]:
```suggestion
  <description>Triggers Tests without Tablets on weekly basis, every Saturday at 10:00
```

Comment (yarongilor) [reply jenkins-pipelines/master-triggers/sct_triggers/no_tablets_weekly_trigger.xml]: fixed

**Review by fruch — APPROVED** [review]
LGTM

File: jenkins-pipelines/master-triggers/sct_triggers/no_tablets_weekly_trigger.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  </triggers>
+  <concurrentBuild>false</concurrentBuild>
+  <builders/>
+  <publishers>
+    <hudson.plugins.parameterizedtrigger.BuildTrigger>
+      <configs>
+        <hudson.plugins.parameterizedtrigger.BuildTriggerConfig>
+          <configs>
+            <hudson.plugins.parameterizedtrigger.PredefinedBuildParameters>
+              <properties>scylla_version=master:latest
```
Comment (fruch) [inline jenkins-pipelines/master-triggers/sct_triggers/no_tablets_weekly_trigger.xml:20]: what about enterprise runs  ?

**Pull request merged**